src/platforms/android/configuration/app-not-respond.mdx

```diff
@@ -47,11 +47,33 @@ You can also specify how long the thread should be blocked before the ANR is rep
 </application>
 ```
 
-## ApplicationExitInfo (v2)
+### ApplicationExitInfo (v2)
 
 This approach reads the [ApplicationExitInfo](https://developer.android.com/reference/android/app/ApplicationExitInfo) API on the next app launch
 and asynchronously sends ANR events to Sentry for each ANR in the history, enriching only the latest one with breadcrumbs, contexts, tags, etc.
 
 The integration reports ANR events with `mechanism:AppExitInfo`.
 
 ![ANR](app-not-respond.png)
+
+## ANR Root Cause Analysis
+
+Sentry performs various root cause analyses to give you insights about why certain ANRs might appear. If a potential root cause is detected, it'll be displayed in a new section underneath the ANR stacktrace. Sentry is capable of detecting the following root causes:
+
+### Performance Issues
+
+If an ANR is connected to one of the Performance Issues detected by Sentry (for example, [File I/O on Main Thread](https://docs.sentry.io/product/issues/issue-details/performance-issues/file-main-thread-io/) or [DB on Main Thread](https://docs.sentry.io/product/issues/issue-details/performance-issues/db-main-thread-io/)), the offending span will be linked to the ANR event, showing you exactly which slow operation contributed to this ANR.
+
+![ANR Root Cause File I/O](anr-rootcause-fileio.png)
+
+### Common Patterns
+
+This detector looks at the stacktrace of the ANR and tries to identify common pitfalls, such as accessing files, assets or heavy resources on the main thread. If an offending stack frame is found, it will be highlighted in the stacktrace alongside some materials on how to prevent this type of ANR:
+
```

Comment:
```suggestion
This detector looks at the stacktrace of an ANR and tries to identify common pitfalls, such as accessing files, assets, or heavy resources on the main thread. If an offending stack frame is found, it'll be highlighted in the stacktrace with some information about how to prevent this type of ANR:

```